tests/helpers/test_entity_platform.py

```diff
@@ -1080,6 +1080,44 @@ async def test_entity_disabled_by_integration(hass):
     assert entry_disabled.disabled_by == er.DISABLED_INTEGRATION
 
 
+async def test_entity_disabled_by_device(hass: HomeAssistant):
+    """Test entity disabled by integration."""
```

Comment:
Stale docstring, please update to explain what the test tests

Reply:
i just swapped integration to device I think it explains it.